src/microsoft/office/license/user/order/microsoft-office-license-user-order.html

```diff
@@ -45,7 +45,6 @@
         </div>
 
         <div data-wizard-step>
-            <p data-translate="microsoft_office_license_order_user_confirm"></p>
```

Comment:
Why did you deleted this line entirely ?

Reply:
The corresponding translation is not present. In old translator, this was just inserting a blank paragraph.

Reply:
👍